woocommerce/wc-admin
# Show empty table when search returns no results

Part of #1451.

Show empty tables when a search returns no results and avoid doing useless API calls.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/52122461-60b9a200-2623-11e9-8358-fc03dd0e88d3.png)

### Detailed test instructions:
- Go to the _Products_ report.
- Make a search that will return no results.
- Verify the table is empty and there are no JS errors.
- Verify there is no call sent to `/wc/v4/reports/products`.
